packages/merge-on-green/src/merge-logic.ts

```diff
@@ -609,32 +542,29 @@ export async function mergeOnGreen(
   labelNames: string[],
   state: string,
   requiredChecks: string[],
+  mogLabel: string,
+  author: string,
   github: OctokitType
 ): Promise<boolean | undefined> {
-  logger.info(`${owner}/${repo} checking merge on green PR status`);
-  const [prInfo, mogLabel, headSha] = await Promise.all([
-    await getPR(owner, repo, pr, github),
-    await hasMOGLabel(owner, repo, pr, labelNames, github),
-    await getLatestCommit(owner, repo, pr, github),
-  ]);
-  logger.info(`required checks: ${requiredChecks}`);
-
-  if (prInfo.state === 'closed') {
-    logger.info(`${owner}/${repo}/${pr} is closed`);
-    return true;
+  const rateLimit = (await github.rateLimit.get()).data.resources.core
+    .remaining;
+  if (rateLimit <= 0) {
```

Comment:
Do we know how quickly quota refills? Should we wait until we have a certain threshold of quota available?

Reply:
It should take an hour to refill! I think if we just keep the PR in the Datastore table, we can just check back in an hour to see if the rate has been replenished (I might be misunderstanding your suggestion here though, feel free to clarify).

Reply:
My question is more around if the quota trickles in -- say we are out of quota, then are given back 3 API calls, but a single m-o-g process takes 5 API calls. We won't be able to complete the m-o-g process and we'll exhaust our quota again.

Reply:
per our conversation, making it ~10 (that's about the API calls it would take to finish)